src/zac/accounts/forms.py

```diff
@@ -105,3 +105,33 @@ def group_key(zaaktype):
             _zaaktypen[catalogus_url] = representations
 
         return _zaaktypen
+
+
+def get_permission_sets_choices():
+    permision_sets = PermissionSet.objects.all()
+    for permision_set in permision_sets:
+        representation = "<strong>{name} - {va}</strong>"
+        zaaktypen = "<br>".join(
+            [format_html(zaaktype.omschrijving) for zaaktype in permision_set.zaaktypen]
+        )
+        if permision_set.zaaktypen:
+            representation = representation + "<br>{zaaktypen}"
+        representation = format_html(
+            representation,
+            name=permision_set.name,
+            va=permision_set.get_max_va_display(),
+            zaaktypen=mark_safe(zaaktypen),
+        )
+        yield permision_set.id, format_html(representation)
```

Comment:
I'm not happy with your current approach yet, I think this is correct though:

```suggestion
def get_permission_sets_choices():
    permision_sets = PermissionSet.objects.all()
    for permision_set in permision_sets:
        representation = "<strong>{name} - {va}</strong>{br}{zaaktypen}"
        zaaktypen = format_html_join(
            mark_safe("<br>"),
            "{}",
            [(zaaktype.omschrijving,) for zaaktype in permision_set.zaaktypen]
        )
        representation = format_html(
            representation,
            name=permision_set.name,
            va=permision_set.get_max_va_display(),
            zaaktypen=zaaktypen,
            br=mark_safe("<br>") if zaaktypen else ""
        )
        yield permision_set.id, format_html(representation)
```

Issues in your new approach:

```py
            [format_html(zaaktype.omschrijving) for zaaktype in permision_set.zaaktypen]
```
-> `zaaktype.omschrijving` is not HTML-escaped here and still an XSS vulnerability, especially because later you do:

```py
            zaaktypen=mark_safe(zaaktypen),
```